docusaurus.config.js

```diff
@@ -143,6 +143,10 @@ module.exports = {
               label: 'Videos',
               to: 'videos/best-practice/jenkins',
             },
+            {
+              to: '/micro',
+              label: 'Microservice',
+            },
```

Comment:
so this should be removed as we already add to footer.

Reply:
Here is the footer link config.